apps/backend/src/libs/modules/config/base-config.module.ts

```diff
@@ -50,7 +50,7 @@ class BaseConfig implements Config {
 					format: Number,
 				},
 			},
-			AWS_S3: {
+			AWS: {
 				AWS_ACCESS_KEY_ID: {
```

Comment:
```suggestion
				ACCESS_KEY: {
```
and others
ACCESS_KEY
REGION